### DISCUSSION TITLE
Will there be a system for reporting bugs?

### DISCUSSION THREAD

**Discussion by lemte** [stoatchat/discussions#746] [Q&A] [answered]
There are some bugs found within the server and I haven't seen where I can report them.

**Comment by Rexogamer** [comment, marked as answer]
bear in mind the team is currently rewriting the web app, so you might want to wait a bit since there won't be any non-critical fixes for the existing app (Revite). that said, bugs can be reported as new issues on the relevant repositories (e.g. revoltchat/backend for an issue with the API).
